Review Miredot license

There is a warning message in the logs during the build of WebAPI that informs about an expired license:

[ERROR] There is a problem with your Miredot license. Please visit http://www.miredot.com for more information or contact support@miredot.com if you have any questions.
The following problems were found with your license: 
 - License key expired on **01 Jan 2025**

--- a/pom.xml
+++ b/pom.xml
@@ -489,7 +489,7 @@
           </execution>
         </executions>
         <configuration>
-          <licence>cHJvamVjdHxvcmcub2hkc2kuV2ViQVBJfDIwMjUtMDEtMDF8ZmFsc2V8LTEjTUN3Q0ZDZkowMkZrVmVSeVlBazZVbTBmNThIbE15SjRBaFEwcVZyRUZuamZMTzJ0KzFtR3E3U3lMMkFESHc9PQ==</licence>
+          <licence>cHJvamVjdHxvcmcub2hkc2kuV2ViQVBJfDIwMzAtMDEtMDF8ZmFsc2V8LTEjTUN3Q0ZDOEFFWEdzU0lISUJiMzBBWC9WazlGcmdPME1BaFI4MVM5bkViQUpXN0tiWE9zR1NpRFlPam5zaXc9PQ==</licence>
           <restModel>
             <httpStatusCodes>
               <httpStatusCode>

